packages/web_benchmarks/lib/src/browser.dart

```diff
@@ -0,0 +1,599 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:async';
+import 'dart:convert' show json, utf8, LineSplitter, JsonEncoder;
+import 'dart:io' as io;
+import 'dart:math' as math;
+
+import 'package:path/path.dart' as path;
+import 'package:meta/meta.dart';
+import 'package:webkit_inspection_protocol/webkit_inspection_protocol.dart';
+
+/// The number of samples used to extract metrics, such as noise, means,
+/// max/min values.
+///
+/// Keep this constant in sync with the same constant defined in `dev/benchmarks/macrobenchmarks/lib/src/web/recorder.dart`.
```

Comment:
Github link to the file?

Reply:
Actually, now that this is the same package, I can move shared constants into a common library.

Reply:
Also, ironically, we _didn't_ keep the two in sync. One was 10, the other 100 lol :)

Reply:
Done.